presets/4.4/other/Tehllama_5in_Race_Tune_1600-2750KV_Freedom_MultiVoltage_3S-4S-5S-6S.txt

```diff
@@ -0,0 +1,702 @@
+#$ FIRMWARE_VERSION: 4.4
+#$ CATEGORY: OTHER
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: 533, Freedom, FreedomSpec, Tune, Spec, Race, Llama, MultiVoltage, Texas, BMS, 3S, 5in, AutoSelect, MM
+#$ AUTHOR: Daniel Appel / Tehllama
+#$ DESCRIPTION: This is a multi-voltage tune built for 5in Race Builds in the 400-575g AUW range. 
+#$ DESCRIPTION: ALL FOUR Tune Profiles are used, with tunes Auto-Selected for 6S, 5S, 4S, and 3S (respectively) 
+#$ DESCRIPTION:
+#$ DESCRIPTION: Select the Motor KV Range from the Drop-Down Menu that most closely matches your craft
+#$ DESCRIPTION: Default is for ~1950KV motors, for example Freedom Spec 2207 1960KV 533/XNOVA Motors
+#$ DESCRIPTION: Special Thanks to Stephen 'Supafly' Wright for Master Slider Calculator assistance
+#$ DESCRIPTION: 
+#$ DESCRIPTION: This tune works best with bidirectional DShot enabled, running RPM and Dynamic Idle Features
+#$ DESCRIPTION: This tune expects ESCs configured to 48kHz PWM Frequency, 23°-27°/MedHigh timing and 0.38 Rampup
+#$ DESCRIPTION: Using more aggressive ESC settings is NOT advised if using higher cell count batteries
+#$ DESCRIPTION: 
+#$ DESCRIPTION: This tune will auto-select profiles based on battery voltage at plugin.
+#$ DESCRIPTION: When 3S packs are plugged in, this will auto-select an aggressive spec racing tune on Profile 3
+#$ DESCRIPTION: At higher motor KV, MotorOutput Limiting is used to scale these to Freedom Spec RPM 
+#$ DESCRIPTION: 3S batteries on craft with <1950KV motors experience very reduced performance
+#$ DESCRIPTION: 
+#$ DESCRIPTION: 6S, 5S, and 4S will run conservative open race tunes 
+#$ DESCRIPTION: Small motor output limits are applied for EXTREME Battery-KV combinations
+#$ DESCRIPTION: Combinations that spin 5" propellers in transonic regimes (i.e. 6S >2200KV and 5S >2650KV)
+#$ DESCRIPTION: 
+#$ DESCRIPTION: !!! Strongly recommend a full chip erase reflash if alternative tunes are desired after installing this preset!!!
+#$ DESCRIPTION: Extensive testing has been done across 83+ builds, however not every craft will run best on this tune.
+#$ WARNING: Use at your own risk.  Verify Correct ESC Settings are used.  
+#$ DISCUSSION: https://github.com/betaflight/firmware-presets/pull/301
+
+## // Update Discussion Link
+
+## Filters - Choose RPM Enabled (Spicy or Non-Spicy) or non-RPM (Super-Safe)
+## Filters - by Profile x4  ++yaw_lowpass_hz
+## Tune - TPA, iTerm, dMin, boost, & AutoCellSelect Parameters
+## Tune - PI, D, I, Dmax, pitch PI/D, FF Offsets
+## KV Selection - Set MM Slider for ALL profiles
+## KV Selection - Set Motor Output Limits where applicable by profile
+## Feature Selection - VBatSagComp, MaxCellVoltage
+
+#$ FORCE_OPTIONS_REVIEW: TRUE
+
+#$ OPTION_GROUP BEGIN: Filters
+#$ OPTION BEGIN (CHECKED): Apply RPM Enabled SAFE Filters to ALL profiles (Requires Bidirectional DShot)
+profile 0
+#$ INCLUDE: presets/4.3/filters/defaults.txt
```

Comment:
Should we just make a presets/4.4/filters/defaults.txt (changed or not)?

Reply:
Discussion in the other thread - @limonspb is correct that there are no changes, so referencing the 4.3 file location doesn't have any ill effects on this